File: DSA/Ayush_Mishra_590014844/Day_32_Question_3.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int arr[n], xor_result = 0;
    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
        xor_result ^= arr[i];
    }

    printf("XOR of all elements is: %d\n", xor_result);
    return 0;
}

File: DSA/Nivedan_590014904/Day_32_Question_3.py
arr = []
n = int(input("Enter the number of elements in array:- "))
for i in range(n):
    val = int(input(f"Enter the {i} element: "))
    arr.append(val)
xor =  arr[0]
for i in range(1,len(arr)):
    xor = xor^arr[i]
print(xor)
